floyd/client/experiment.py

```diff
@@ -40,6 +42,19 @@ def create(self, experiment_request):
         return response.json().get("id")
 
     def delete(self, id):
-        self.request("DELETE",
-                     "{}{}".format(self.url, id))
-        return True
+        try:
+            self.request("DELETE",
+                         "{}{}".format(self.url, id))
+            floyd_logger.info("Experiment {}: Deleted".format(id))
+            return True
+        except NotFoundException as e:
+            floyd_logger.info(
+                    ("Experiment {}: ERROR! A deletable experiment with this "
+                     "id was not found. Make sure you have the correct id and "
+                     "that the experiment is not "
```

Comment:
It is unfortunate that the server returns 404 if the experiment is queued or running. It should ideally return 400.